pam/integration-tests/testdata/TestCLIIntegration/golden/deny_authentication_if_max_attempts_reached

```diff
@@ -265,8 +265,10 @@ invalid password, should be goodpass
 > ./pam_authd socket=/tmp/pam-cli-tests.sock
 Gimme your password
 >
-Auth return: Authentication failure: invalid password, should be goodpass
-Acct mgmt return: The return value should be ignored by PAM dispatch
+PAM ERROR: invalid password, should be goodpass
```

Comment:
It seems here the error is stuttered, no?

Reply:
So, these errors are now printed because of a syntax error that has been fixed in pam.go in this commit: https://github.com/ubuntu/authd/pull/155/commits/78c6ff74584261cade99b93706ca267668119799#diff-02a489eb5b68908dc93afc83b86714473c3cebe6a4684c7d23ba3af1dbe5287e

Now the PAM conversation is handled on the client side and so we also print the errors that are sent through PAM protocol to the user.

I'm not sure if that's something we want all the times, once we're entered in the interactive mode, but we can avoid sending them in such case. The important thing IMHO is that we should definitely send them on final errors or when no UI has been sent since that's the only information that is for sure sent to the user.